build/dockerfiles/rhel.install.sh

```diff
@@ -8,7 +8,36 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all && \
+architecture=""
+case $(uname -m) in
+       x86_64) architecture="amd64" ;;
+       s390x) architecture="s390x";;
+esac
+if [ $architecture == "amd64" ]; then \
+       microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all; \
+elif [ $architecture == "s390x" ]; then \
+       microdnf install -y git findutils bash wget yum gzip tar python3-six python3-pip && microdnf -y clean all && \
+       microdnf install -y gcc gcc-c++ make automake; \
+       wget https://github.com/stedolan/jq/releases/download/jq-1.5/jq-1.5.tar.gz; \
+       tar xvzf jq-1.5.tar.gz; \
+       cd jq-1.5 || exit; \
+       ./configure && make && make install; \
```

Comment:
Rather than building from source could we use Fedora version?

https://rpmfind.net/linux/rpm2html/search.php?query=jq&submit=Search+...&system=&arch=s390x